realm/src/main/java/io/realm/exceptions/RealmEncryptionNotSupportedException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.exceptions;
+
+/**
+ * On some devices (HTC One X for example), for some reason, the system doesn't pass the right
```

Comment:
My main point is that this seems very rare and therefore users needs to have an idea about how rare this is and that they should expect encryption to work in almost all cases. Do we have some stat's from the users involved? How many devices in % of their total were affected?